shell/browser/api/electron_api_web_contents.cc

```diff
@@ -2064,11 +2064,11 @@ void WebContents::CopyImageAt(int x, int y) {
     host->CopyImageAt(x, y);
 }
 
+#if !defined(OS_MACOSX)
```

Comment:
The `web_contents()->Focus()` should still be called on macOS otherwise the specified WebContents may not be focused when the window has multiple WebContents.